src/lib/components/error-pages/ErrorPage500.component.tsx

```diff
@@ -48,7 +48,7 @@ function ErrorPage500({
   return (
     <ErrorPageContainer className="sc-error-page500" {...rest}>
       <Row>
-        <WarningIcon className="fas fa-exclamation-triangle fa-2x" />
+        <Icon name="Exclamation-triangle" size="2x" color="statusWarning" />
```

Comment:
@Cuervino shall it be replaced by the circle icon ?

Reply:
I thought harmonization is in second step ?

Reply:
@Cuervino

Reply:
Let's do it at a later stage in that case